resources/basics.py

```diff
@@ -54,18 +54,46 @@
 }
 
 
+def get_release_file(file_path: str, version: str = CURRENT_RELEASE) -> str:
+    """
+    Tries to get the desired file from the corresponding release version on the google cloud.
+    If the file is not found for the desired release version, falls back on previous versions.
+
+    :param  str file_path: Desired file path, with {0} as placeholder(s) for the version number
+    :param str version: Desired file version
+    :return: Path for closest version of the file available
+    """
+    if hl.hadoop_exists(file_path.format(version)):
+        return file_path.format(version)
+    else:
+        for v in range(RELEASES.index(version)-1, -1, -1):
+            if hl.hadoop_exists(file_path.format(RELEASES[v])):
+                print("WARN: Resource {} could not be found for gnomAD release version {}.\n Loading gnomAD release version {} of the file. ({})".format(
```

Comment:
Might look nicer with an f-string, but that's up to you

Reply:
Because of the nested `format` statements I feel like `format` is a little more readable in this case...

Reply:
You could still nest them:
```
print("WARN: Resource {file_path.format(version)} could not be found for gnomAD release version {version}.\n Loading gnomAD release version {RELEASES[v]} of the file. ({file_path.format(RELEASES[v])})")
```
but if you disagree, that's fine.